MaterialLib/MPL/MediumAuxiliaryFunction.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * File:   MediumAuxiliaryFunction.h
+ *
+ * Created on July 29, 2019, 2:48 PM
+ */
+
+#pragma once
+
+#include "MaterialLib/MPL/PropertyType.h"
+
+namespace MaterialPropertyLib
+{
+template <int GlobalDim>
+Eigen::Matrix<double, GlobalDim, GlobalDim> localTensor(
+    MaterialPropertyLib::PropertyDataType const& values)
+{
+    if (boost::get<double>(&values))
```

Comment:
I suggest to rewrite the ifs with visitor, as described in [boost docu](https://www.boost.org/doc/libs/1_70_0/doc/html/variant/tutorial.html#variant.tutorial.basic)

Reply:
Done.